test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -97,20 +104,26 @@ private static void printToErr(String s) {
     }
 
     protected static class GradleMessageBuilder extends ReproduceErrorMessageBuilder {
+        private static final Logger logger = LogManager.getLogger(GradleMessageBuilder.class);
 
         public GradleMessageBuilder(StringBuilder b) {
             super(b);
         }
 
         @Override
         public ReproduceErrorMessageBuilder appendAllOpts(Description description) {
-            super.appendAllOpts(description);
-
-            if (description.getMethodName() != null) {
-                //prints out the raw method description instead of methodName(description) which filters out the parameters
-                super.appendOpt(SYSPROP_TESTMETHOD(), "\"" + description.getMethodName() + "\"");
+            RandomizedContext ctx = null;
+            try {
```

Comment:
I don't think copying appendAllOpts is necessary. We already override appendOpt, and filter away some properties we don't want to print.

Reply:
Good call. I've just added the test class to the list of filtered items in `appendOpt()`.